osm-pds.sh

```diff
@@ -25,6 +25,10 @@ function mirror() {
 
   if [[ "$input" =~ ^s3:// ]]; then
     aws s3 cp $input $output
+
+    # Attempt to also copy the related md5 file
+    aws s3 cp $input.md5 $output.md5 || echo "Unable to copy md5"
```

Comment:
This is done in the Lambda function, as the existence of the MD5 is used as a marker to indicate that mirroring jobs have already been queued.

Reply:
I think the Lambda function creates the md5 in S3 when doing a mirror from the planet.osm.org site, but I don't think it was being set when doing an S3-to-S3 mirror, basically a copy/rename of the latest. I'll take another look at it and confirm.

Reply:
Yup, you're right. md5s only exist for PBFs (no easy way to generate one for the ORC versions, since it's never fully on a filesystem).